sys/dev/e1000/if_em.c

```diff
@@ -1093,9 +1093,6 @@ em_if_attach_pre(if_ctx_t ctx)
 		}
 	}
 
-	/* Disable ULP support */
-	e1000_disable_ulp_lpt_lp(hw, true);
```

Comment:
This is redundant, it is called in ich8lan setup via  e1000_setup_init_funcs() above.